Docs 7741 clean up deploying to cloudhub hn (#344)
mulesoft/docs-runtime-manager: modules/ROOT/pages/cloudhub.adoc

=== Integration with Anypoint Studio

> With xref:studio::index.adoc[Anypoint Studio (Studio)] you can quickly build integration applications and deploy to CloudHub with just a few clicks, without leaving Studio. See xref:deploying-to-cloudhub.adoc#deploy-an-application-from-studio[Deploy an Application from Studio] for more information.

Reviewer: ```suggestion
Using xref:studio::index.adoc[Anypoint Studio (Studio)], you can build integration applications and deploy them to CloudHub with just a few clicks. See xref:deploying-to-cloudhub.adoc#deploy-an-application-from-studio[Deploy an Application from Studio] for more information.
```

Reviewer: But see below....

> Once your application is deployed to CloudHub's multi-tenant runtime, access it like any other application deployed through the platform, by http://anypoint.mulesoft.com[signing into] Anypoint Platform, and then going to Runtime Manager.

Reviewer: ...or consider this, without the xref above:
```suggestion
Using xref:studio::index.adoc[Anypoint Studio (Studio)], you can build integration applications and deploy them to CloudHub's multitenant runtime with just a few clicks. You can then access them like any other application deployed through the platform, by http://anypoint.mulesoft.com[signing in to] Anypoint Platform and then navigating to Runtime Manager.
```
